--silent doesn't work

the following example is from readme.md
```
{
  "scripts": {
    "clean": "shx --silent rm -rf build dist && shx echo Done",
    "example": "shx --silent ls fakeFileName"
  }
}
``` 

output:
```
npm ERR! Lifecycle script `example` failed with error: 
npm ERR! Error: command failed
```

--- a/README.md
+++ b/README.md
@@ -59,6 +59,7 @@ sub
 $ shx rm -r sub                 # options work as well
 
 $ shx --silent ls fakeFileName  # silence error output
+$ shx --silent ls fakeFileName || shx true  # silence error output and ignore the exit status
 
 $ shx --negate test -d dir      # Negate status code output (e.g., failed commands will now have status 0)
 ```

